lib/src/domain/model/article.dart

```diff
@@ -1,17 +1,42 @@
-import 'package:json_annotation/json_annotation.dart';
+import 'package:allthenews/src/data/response/article_response.dart';
 
-part 'article.g.dart';
-
-@JsonSerializable()
 class Article {
   final int id;
 
+  final String url;
+
+  final String updated;
+
+  final String authorName;
+
   final String title;
 
-  Article(this.id, this.title);
+  final String abstract;
+
+  final String thumbnail;
+
+  Article({
+    this.id,
```

Comment:
moglibyśmy ustalić, żeby obiekty domenowe miały w konstruktorze @required, bo w sumie tutaj nie jest wymuszone podanie wszystkich pól tylko po to, żeby móc skorzystać z named arguments, a dodanie tego @required ma zalety obu podejść i konstruktora z nazwanymi parametrami i unikania możliwości zbudowania wadliwego obiektu. Jako, że to jest warstwa domeny to required powinien być zaimportowany z pakietu `package:meta/meta.dart`, żeby nie było zależności flutterowych w domenie - napisałem, bo domyślnie IDE importuje z flutter/material lub cupertino :)